lib/services/apiAppManagement/README.md

```diff
@@ -0,0 +1,33 @@
+# Microsoft Azure SDK for Node.js - ApiApp Management
+
+This project provides a Node.js package that makes it easy to manage Microsoft ApiApp. Right now it supports:
+- **Node.js version: 0.6.15 or higher**
+- **API version: 2015-01-14**
+
+## Features
+
+ - TODO
```

Comment:
@yugangw-msft Exactly. Also, I copied this file pretty much verbatim from the websiteManagement2 project. We're more preview then they are. :-)